pkg/tests/integration_test.go

```diff
@@ -133,3 +133,13 @@ func TestLocalPlugin(t *testing.T) {
 		},
 	})
 }
+
+//nolint:paralleltest // uses parallel programtest
+func TestResourceOrderingWithDefaultProvider(t *testing.T) {
```

Comment:
Does this actually fail without the fix? Doesn't it just use the engine default for the alb and so still preview ok?

Reply:
nope!

Reply:
Nope!